.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"google_sign_in","path":"C:\\\\src\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\google_sign_in-5.0.2\\\\","dependencies":[]}],"android":[{"name":"google_sign_in","path":"C:\\\\src\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\google_sign_in-5.0.2\\\\","dependencies":[]}],"macos":[],"linux":[],"windows":[],"web":[{"name":"google_sign_in_web","path":"C:\\\\src\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\google_sign_in_web-0.10.0\\\\","dependencies":[]}]},"dependencyGraph":[{"name":"google_sign_in","dependencies":["google_sign_in_web"]},{"name":"google_sign_in_web","dependencies":[]}],"date_created":"2021-05-01 05:17:24.975951","version":"2.0.6"}
```

Comment:
As the file states, can you not include this file in the PR? :)

Reply:
Sorry, looks like there're missing files in my .gitignore file.